src/algorithms/sdp.jl

```diff
@@ -100,7 +100,7 @@ end
 function sd_objective(objective0, sd_constraints, c::AbstractArray)
     function _objective(args)
         target = objective0(args)
-        barrier = sum(c .* -logdet.(sd_constraints.fs.(args)))
+        barrier = sum(c .* -logdet.(map(f -> f(args), sd_constraints.fs)))
```

Comment:
@mohamed82008 What makes this different? I found weird thing before I went to bed yesterday, not sure if that's a bug. That if two `MvNormal` applied to one optimization then the first one is not well optimized, is this the reason?